src/Storages/MergeTree/MutateTask.cpp

```diff
@@ -1029,6 +1049,42 @@ void finalizeMutatedPart(
     new_data_part->default_codec = codec;
 }
 
+/// For alter table modify columns, check which option is set and which column is changed.
```

Comment:
(cosmetic: function `getSecondaryIndicesOnColumnAlterModifyOptions` (l. 1053) is very "far" away from the code that uses it (l. 1520).)